tests/mocha/test.js

```diff
@@ -32,6 +32,17 @@ describe('Mocha smoke test', () => {
         assert.equal(el.$('.selector-2').isExisting(), true)
     })
 
+    it('should waitUntil', () => {
```

Comment:
This smoke test is not clear to me. Can we create a more real world scenario where the first 3 request fail (e.g. for isExisting) and then the 4th works and we check if the condition was executed 4 times and the return value of waitUntil is true?

Reply:
Sorry to say have no better ideas to verify if wait until command promises are resolved. 
However I can rename the test to reflect what it actually does, is this fine?

Reply:
argh, no, I can't use isExisting, because it is element call, and I need browser call, anyway will tune test a bit

Reply:
@christian-bromann take a look

Reply:
I can't say it is more meaningful but at least test name reflects what it does